### PR TITLE
Made fixes to customer pages

### PR DESCRIPTION
**Pull request by emmaakachukwu**
- [ ] Create customer
- [ ] Edit, view and delete customer
- [ ] Dummy data removed
- [ ] International Phone number standard fully implemented with validation
- [ ] No broken links

> Test: https://acemycustomer.herokuapp.com/

a

b

### CODE REVIEW

File: app/Http/Controllers/CustomerController.php [hunk 1]
```diff
@@ -114,12 +125,12 @@ public function create()
...
-        $url = env('API_URL', 'https://dev.api.customerpay.me') . '/customer/new/';
+        $url = $this->host.'/customer/new/';
 
         if ($request->isMethod('post')) {
             $request->validate([
-                'store_name' => 'required',
-                'phone_number' =>  'required | min:8 | max:15',
+                'store_id' => 'required',
+                'phone_number' =>  ['required', 'min:6', 'max:16',  new NoZero, new DoNotPutCountryCode],
                 'name' => 'required | min:5 | max:30',
```
Comment (Jeremiahiro) [inline app/Http/Controllers/CustomerController.php:134]: I don't understand this ` new NoZero, new DoNotPutCountryCode],`

File: resources/views/backend/customer/index.blade.php [hunk 2]
```diff
@@ -462,66 +453,66 @@ class="avatar-sm rounded-circle" alt="Shreyu" /></td>
...
-                                                            onclick="
-                                                                    if(confirm('Are you sure You want to delete this user'))
-                                                                    {event.preventDefault(); document.getElementById('delete-form-{{ $response[$i]->_id }}').submit();}
-                                                                    else{
-                                                                    event.preventDefault();
-                                                                }"> Delete </a>
+                                            <tr>
+                                                <td>{{$i + 1}}</td>
+                                                <td>{{isset($response[$i]->name) ? ucfirst($response[$i]->name) : 'Not available'}}</td>
+                                                <td>{{isset($response[$i]->phone_number) ? $response[$i]->phone_number : 'Not available'}}</td>
```
Comment (Jeremiahiro) [inline resources/views/backend/customer/index.blade.php:473]: it's either there's a value or not, you cannot set `not available` its seen as dummy data

Comment (emmaakachukwu) [reply resources/views/backend/customer/index.blade.php:473]: Okay. will change that right away

Comment (emmaakachukwu) [reply resources/views/backend/customer/index.blade.php]: @Jeremiahiro updated

**Pull request merged**